tmva/tmva/inc/TMVA/CvSplit.h

```diff
@@ -0,0 +1,180 @@
+// @(#)root/tmva $Id$
+// Author: Kim Albertsson
+
+/**********************************************************************************
+ * Project: TMVA - a Root-integrated toolkit for multivariate data analysis       *
+ * Package: TMVA                                                                  *
+ * Class  : CvSplit                                                               *
+ * Web    : http://root.cern.ch                                                   *
+ *                                                                                *
+ * Description:                                                                   *
+ *                                                                                *
+ * Authors (alphabetical):                                                        *
+ *      Kim Albertsson <kim.albertsson@cern.ch> - CERN/LTU                        *
+ *      Sergei Gleyzer <sergei.gleyzer@cern.ch> - CERN, Switzerland               *
+ *      Lorenzo Moneta <Lorenzo.Moneta@cern.ch> - CERN, Switzerland               *
+ *                                                                                *
+ * Copyright (c) 2005-2017:                                                       *
+ *      CERN, Switzerland                                                         *
+ *      ITM/UdeA, Colombia                                                        *
```

Comment:
Same as above, copyright to Rene and Fons (same for other files as well).

Reply:
@amadio, this comment is not valid! Please ignore it

Reply:
Is it an exception for TMVA? For ROOT in general I was told that copyright must be assigned to Rene and Fons.